syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode
{
    int32 errcode = 1;
    bytes errmsg = 2;
}

message GetFriendListRequest
{
    uint32 userid = 1;
}

message GetFriendListResponse
{
    ResultCode result = 1;
    repeated bytes friends = 2;  
}

/// 
/// UserServiceRpc provides user-related RPC methods such as login.
///
service FriendServiceRpc 
{   
    /// GetFriendList returns the list of friends for the specified user.
    rpc GetFriendList(GetFriendListRequest) returns(GetFriendListResponse);
}